ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -59,3 +95,15 @@ def handle(self, *args, **options):
                   nb_media=len(package.medias)
                   )
               )
+
+        if not options['dry_run']:
+            print("\nYaml to use is: \n")
+            yaml_metadata = generate_catalog_yaml_metadata(
+                package_path,
+                'zipped-medias',
+                options['package_id'],
+                options['name'],
+                options['description'],
+                options['language']
```

Comment:
... but if you add the URL as a cli argument, then you probably should put it in the YAML. 😉 

(but again, I just wouldn't bother)